graphql/resolve/resolver.go

```diff
@@ -324,12 +325,16 @@ func (r *RequestResolver) Resolve(ctx context.Context, gqlReq *schema.Request) *
 	}
 
 	if glog.V(3) {
-		b, err := json.Marshal(gqlReq.Variables)
-		if err != nil {
-			glog.Infof("Failed to marshal variables for logging : %s", err)
+		// don't log the introspection queries they are sent too frequently
+		// by GraphQL dev tools
+		if !op.IsQuery() || !strings.HasPrefix(op.Queries()[0].Name(), "__") {
```

Comment:
The second check in the OR block could also have compared the QueryType to make it a bit clearer.